benchopt/runner.py

```diff
@@ -205,6 +205,36 @@ def get_next(x): return max(x / rho, MIN_TOL)
     return curve, status
 
 
+def get_callback(curve_one_rep, cb_status, objective, solver,
+                 meta_rep, deadline):
+    # make documentation
+    info = get_sys_info()
```

Comment:
Looking back at the code, I think I would rather define `curve_one_rep` and `cb_status` in this function as it will make it clearer what it is.
Sorry for the change of mind ^^